test/runtime/assertionHelpers.go

```diff
@@ -39,3 +39,16 @@ func ExpectEndpointSummary(vm *helpers.SSHMeta, policyEnforcementType string, nu
 	ExpectWithOffset(1, err).Should(BeNil(), "error getting endpoint summary")
 	ExpectWithOffset(1, endpoints[policyEnforcementType]).To(Equal(numWithPolicyEnforcementType), "number of endpoints with %s=%s does not match", helpers.PolicyEnforcement, policyEnforcementType)
 }
+
+// ExpectCiliumReady asserts that cilium status is ready
+func ExpectCiliumReady(vm *helpers.SSHMeta) {
+	err := vm.WaitUntilReady(100)
+	Expect(err).To(BeFalse(), "Cilium-agent cannot be started")
```

Comment:
Assertions should use `ExpectWithOffset` or some variant of it so that we can see where this helper failed in line with the test in which it is called.

Reply:
good catch!